### PR TITLE
ProgressBar - update color of progressbar-status

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

**Review by marker-dao — DISMISSED** [review]

File: packages/devextreme/scss/widgets/fluent/progressBar/_index.scss [hunk 1]
```diff
@@ -60,7 +60,7 @@
   }
 
   .dx-progressbar-status {
-    color: $base-foreground-disabled;
+    color: $base-text-color;
```
Comment (EugeniyKiyashko) [inline packages/devextreme/scss/widgets/fluent/progressBar/_index.scss]: Progressbar status should have $base-foreground-disabled color in disabled state. It's correct. Please, revert this change.

Comment (alexanderPolosatov) [reply packages/devextreme/scss/widgets/fluent/progressBar/_index.scss]: fixed

**Review by EugeniyKiyashko — APPROVED** [review]
